chroma-manager/Makefile

```diff
@@ -208,6 +208,13 @@ rpms: cleandist tarball
 	rm -rf _topdir
 	echo "jenkins_fold:end:Make Manager RPMS"
 
+install_requirements: requirements
+	echo "jenkins_fold:start:Install Python requirements"
+	pip install --upgrade pip;                              \
+	pip install --upgrade setuptools;                       \
+	pip install -Ur requirements.txt
+	echo "jenkins_fold:end:Install Python requirements"
+
 requirements: requirements.txt
```

Comment:
don't think it's necessary to be  repush for this , but I will change if repush for something